common/ir.go

```diff
@@ -63,12 +69,7 @@ func Multiaddress(n []interop.PublicKey, committee bool) []byte {
 		threshold = len(n)/2 + 1
 	}
 
-	keys := []interop.PublicKey{}
```

Comment:
why did we have such a transformation? and why it becomes useless now?

Reply:
As far as I remember, `neo.GetCommittee()` returned `[][]byte` instead of `[]interop.PublicKey{}` so we had to do these, but I am not 100% sure.